File: 32B.py
s = input()
ss = ''
ans = ''
for i in s:
    if ss == '':
        if i == '.':
            ans += '0'
        else:
            ss+=i

    elif ss == '.':
        ans += '0'
        ss = ''
    elif ss == '-':
        if i == '.':
            ans += '1'
            ss = ''
        else:
            ans += '2'
            ss = ''

print(ans)

File: 32B.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    string s;
    cin >> s;
    string ss, ans;

    for (auto i : s) {

        if (ss.empty()) {
            if (i == '.') {
                ans += '0';
            } else {
                ss += i;  // likely '-'
            }
        }
        else if (!ss.empty() && ss == "-") {
            if (i == '.') {
                ans += '1';
                ss = "";
            } else if (i == '-') {
                ans += '2';
                ss = "";
            }
        }
    }

    cout << ans << endl;
    return 0;
}
